include/mp/proxy-types.h

```diff
@@ -726,6 +726,13 @@ kj::Promise<void> serverInvoke(Server& server, CallContext& call_context, Fn fn)
                 MP_LOG(*server.m_context.loop, Log::Debug) << "IPC server send response #" << req << " " << TypeName<Results>();
                 MP_LOG(*server.m_context.loop, Log::Trace) << "response data: "
                     << LogEscape(call_context.getResults().toString(), server.m_context.loop->m_log_opts.max_chars);
+            }, [&server, req](::kj::Exception&& e) {
```

Comment:
isn't the local `server_capture` being captured by reference problematic if `invoke` is being called async? Possible that's smashing the stack here?

(Maybe I'm way off, these tangled callbacks a breaking my brain).

But I think I'd expect this to be `return ReplaceVoid([server_context = std::move(server_context)]()...`